src/adapters/swagger20.es6

```diff
@@ -157,8 +164,11 @@ export function parse({ source }, done) {
         schemaAsset.attributes.contentType = 'application/schema+json';
       }
 
+      // Currently, default responses are not supported in API Description format
+      let relevantResponses = _.omit(methodValue.responses, 'default');
```

Comment:
I don't know what `default` responses in swagger are and why Blueprint doesn't support them. Can you explain it for a me a bit briefly?